.circleci/integration-tests/master_dag.py

```diff
@@ -93,12 +93,19 @@ def prepare_dag_dependency(task_info, execution_time):
         {"s3_sensor_dag": "example_s3_sensor"},
         {"redshift_sql_dag": "example_async_redshift_sql"},
         {"redshift_cluster_mgmt_dag": "example_async_redshift_cluster_management"},
-        {"emr_sensor_dag": "example_emr_sensor"},
```

Comment:
if we are clubbing all the EMR related DAG's , let's put a comment here mentioning the same. looks like you are doing same thing for amazon related DAG. Let's mention these as comments so that anyone who changes this script in future can understand.